### DISCUSSION TITLE
trying to install themaxmarchuk/tailwindcss-colors.nvim

### DISCUSSION THREAD

**Discussion by dmitrix11** [NvChad/NvChad#1724] [Q&A] [answered]
Hi, I'm new here and trying to figure out how to activate themaxmarchuk/tailwindcss-colors.nvim on the buffer.
It works with :TailwindColorsToggle, but I'd like to load it automatically when tailwind-language-server is active/running.

Thank you.

**Comment by siduck** [comment, marked as answer]
hi @dmitrix11 , why use another plugin when we have this feature in our plugin itself?
![image](https://user-images.githubusercontent.com/59060246/209456929-4850079c-ee72-4760-8778-7580a7ffff90.png)

**Reply by dmitrix11** [reply]
Thank you. Missed it. Enjoying your project a LOT!!!

**Reply by dmitrix11** [reply]
What would be the best way to override/enable that option? Thank you again.

**Reply by siduck** [reply]
@dmitrix11 try 

```lua
["NvChad/nvim-colorizer.lua"] = {
  override_options = {
      user_default_options = { tailwind = true }
   }
}
```

**Reply by dmitrix11** [reply]
PS. true didn't work for some reason; but "lsp" did.

**Reply by siduck** [reply]
hmm @Akianonymus will check it in few days as he maintains that plugin, till then just make tailwind-colors.nvim plugin to load onattach, an example 

```lua
local on_attach = require("plugins.configs.lspconfig").on_attach
local capabilities = require("plugins.configs.lspconfig").capabilities

local lspconfig = require "lspconfig"
local servers = { "html", "cssls", "clangd", "jsonls", "tsserver", "tailwindcss", "bashls" }

for _, lsp in ipairs(servers) do
  lspconfig[lsp].setup {
    on_attach = on_attach,
    capabilities = capabilities,
  }

  if lsp == "tailwindcss" then
    lspconfig[lsp].setup {
      on_attach = function(_, bufnr)
        require("tailwindcss-colors").buf_attach(bufnr)
      end,
      capabilities = capabilities,
    }
  end
end
```
